admission-webhook/make/image.mk

```diff
@@ -1,17 +1,38 @@
 # must stay consistent with the go version defined in .travis.yml
 GO_VERSION = 1.20
-DOCKER_BUILD = docker build . --build-arg GO_VERSION=$(GO_VERSION) --build-arg VERSION=$(TAG)
+BUILD_ARGS = --build-arg GO_VERSION=$(GO_VERSION) --build-arg VERSION=$(TAG) 
+DOCKER_BUILD = docker build . $(BUILD_ARGS)
+
+AMD64_ARGS = --build-arg GOARCH=amd64 --build-arg ARCH=amd64 --platform=linux/amd64
+ARM64_ARGS = --build-arg GOARCH=arm64 --build-arg ARCH=arm64 --platform=linux/arm64
+BUILDX_BUILD = docker buildx build . $(BUILD_ARGS) --provenance=false --sbom=false
+
 
 .PHONY: image_build_dev
 image_build_dev:
 	$(DOCKER_BUILD) -f dockerfiles/Dockerfile.dev -t $(DEV_IMAGE_NAME)
 
-.PHONY: image_build
-image_build:
-	$(DOCKER_BUILD)  -f dockerfiles/Dockerfile -t $(WEBHOOK_IMG):$(TAG)
-	docker tag $(WEBHOOK_IMG):$(TAG) $(WEBHOOK_IMG):latest
+.PHONY: create_buildx_builder
+create_buildx_builder:
+	docker buildx create --name img-builder --platform linux/amd64 --use
+
+.PHONY: remove_image_builder
+remove_image_builder:
+	docker buildx rm img-builder || true
+
+image_build: remove_image_builder create_buildx_builder image_build_int remove_image_builder
+
+.PHONY: image_build_int
+image_build_int:
+	$(BUILDX_BUILD) --load $(AMD64_ARGS) -f dockerfiles/Dockerfile -t $(WEBHOOK_IMG):$(TAG) -t $(WEBHOOK_IMG):latest
```

Comment:
do you need to have a line that does this for `ARM64_ARGS`

Reply:
the `image_build` targers only build and amd64 image are are intended to be used during integration tests.
the `image_build_and_push` targets build both amd64 and arm64 images and we do use `ARM64_ARGS` in there.

I added some comment to clarify this.
Let me know if this makes sense